server/storage/memory.go

```diff
@@ -158,6 +196,25 @@ func (st *MemStorage) Delete(gun string) error {
 	return nil
 }
 
+// GetChanges returns a []Change starting from but excluding the record
+// identified by changeID. In the context of the memory store, changeID
+// is simply an index into st.changes. The ID of a change, and its index
+// are equal, therefore, we want to return results starting at index
+// changeID+1 to match the exclusivity of the interface definition.
+func (st *MemStorage) GetChanges(changeID string, pageSize int, filterName string, reversed bool) ([]Change, error) {
+	id, err := strconv.ParseInt(changeID, 10, 32)
```

Comment:
`pageSize` can be `ParseUint`, but the `changeID` can be `-1` to indicate "start from the most recent change", in which case, I'm doing to add a check to ensure if the `changeID` is `-1` the user has also requested the results to be reversed, otherwise it doesn't make sense.